plugins/commands/compare/src/main/kotlin/CompareCommand.kt

```diff
@@ -70,6 +71,12 @@ class CompareCommand : OrtCommand(
     ).enum<CompareMethod>()
         .default(CompareMethod.TEXT_DIFF)
 
+    private val contextSize by option(
+        "--context-size", "-s",
```

Comment:
The mentioned tools use upper-case `-C`, however. I'm now using that as well.

Reply:
They use both
```
diff --help | grep context
  -c, -C NUM, --context[=NUM]   output NUM (default 3) lines of copied context
```

Reply:
Not on my Linux box:

```
grep --help | grep -- "-[cC],"
  -c, --count               print only a count of selected lines per FILE
  -C, --context=NUM         print NUM lines of output context
```

Reply:
Ah, sorry, you were looking at `diff` and me at `grep`... I guess `diff` makes more sense as a reference, but since `-C` seem to be the "lowest common denominator", shall we stick to it?